src/main/java/mate/controller/driver/DeleteDriverController.java

```diff
@@ -0,0 +1,23 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/delete")
+public class DeleteDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long id = Long.valueOf(req.getParameter("id"));
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        driverService.delete(id);
+    }
```

Comment:
After deleting the driver, you should redirect the user to another page. Use 'resp.sendRedirect()' to redirect the user to a relevant page.